.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,178 @@
+#  docker-build-dse-ubi:
+#    name: Build DSE UBI Docker image
+#    needs: docker-build-dse-ubuntu
+#    runs-on: ubuntu-latest
+#    steps:
+#      - name: Check out source code
+#        uses: actions/checkout@v3
+#      - name: Setup Maven settings file
+#        run: |
+#          mkdir -p ~/.m2
+#          cat <<EOF > ~/.m2/settings.xml
+#          <settings>
+#            <servers>
+#              <server>
+#                <id>artifactory-snapshots</id>
+#                <username>${{ secrets.ARTIFACTORY_USERNAME }}</username>
+#                <password>${{ secrets.ARTIFACTORY_PASSWORD }}</password>
+#              </server>
+#              <server>
+#                <id>artifactory-releases</id>
+#                <username>${{ secrets.ARTIFACTORY_USERNAME }}</username>
+#                <password>${{ secrets.ARTIFACTORY_PASSWORD }}</password>
+#              </server>
+#            </servers>
+#          </settings>
+#          EOF
+#          cp ~/.m2/settings.xml settings.xml
+#      - name: Setup Docker meta
+#        id: meta
+#        uses: docker/metadata-action@v4
+#        with:
+#          images: datastax/dse-mgmtapi-6_8
+#          tags: type=sha,prefix=dse68-ubi7-
+#      - name: Setup Docker DSE inherit meta
+#        id: dsemeta
+#        uses: docker/metadata-action@v4
+#        with:
+#          images: datastax/dse-mgmtapi-6_8
+#          tags: type=sha,prefix=dse68-jdk8-
+#      - name: Set up QEMU
+#        uses: docker/setup-qemu-action@v2
+#      - name: Set up Docker Buildx
+#        uses: docker/setup-buildx-action@v2
+#      - name: Login to DockerHub
+#        uses: docker/login-action@v2
+#        with:
+#          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+#          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+#      - name: Set outputs
+#        id: vars
+#        run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+#      - name: Download Docker registry data from build job
+#        uses: actions/download-artifact@v3
+#        with:
+#          name: docker-registry-data
+#          path: ${{ env.REGISTRY_PATH }}
+#      - name: Start Docker registry
+#        run: |
+#          docker run --rm --detach --publish 5000:5000 \
+#            --volume ${REGISTRY_PATH}:/var/lib/registry \
+#            --name registry registry:2
+#          sleep 10
+#      - name: Build DSE 6.8-ubi7
+#        id: docker_build
+#        uses: docker/build-push-action@v3
+#        with:
+#          file: dse-68/Dockerfile.ubi7
+#          build-args: VERSION=dse68-jdk8-${{ steps.vars.outputs.sha_short }}
+#          context: .
+#          push: false
+#          tags: ${{ steps.meta.outputs.tags }}
+#          platforms: linux/amd64,linux/arm64
+#          target: dse68
+#          cache-from: |
+#            type=local,src=${{ env.CACHE_PATH }}
+#            ${{ steps.dsemeta.outputs.tags }}
+#          cache-to: type=local,dest=${{ env.CACHE_PATH }}
```

Comment:
This entire section(for DSE UBI images) is commented out because I haven't been able to find a way to build the DSE Ubuntu images in the previous job and make the images available here. The UBI images pull from the Ubuntu based images, and I can't figure out how to get the Docker Buildx cache transferred from the previous job into this job so that the UBI builds don't try to pull from Docker.

Reply:
@adejanovski If this limitation is ok for now, I think this is good to go.